### PR TITLE
Can query API with IPAddress instead of only string

### PR DESCRIPTION
**Pull request by gturri**

### CODE REVIEW

File: GeoIP2/WebServiceClient.cs [hunk 1]
```diff
@@ -148,6 +148,16 @@ internal InsightsResponse Insights(string ipAddress, IRestClient restClient)
         /// <summary>
         /// Returns an <see cref="CountryResponse"/> for the specified ip address.
         /// </summary>
+        /// <param name="i">The ip address.</param>
```
Comment (oschwald) [inline GeoIP2/WebServiceClient.cs]: Minor nit, but this doesn't match the param name.

**Comment by oschwald** [comment]
Thanks. This looks good. Eventually I'd like to do the same with the [underlying database reader](https://github.com/maxmind/MaxMind-DB-Reader-dotnet/blob/master/MaxMind.Db/Reader.cs#L167) to avoid the double object creation. I'm not sure why we originally went with a string representation in this API.

File: GeoIP2/WebServiceClient.cs [hunk 2]
```diff
@@ -169,6 +179,16 @@ internal CountryResponse Country(string ipAddress, IRestClient restClient)
         /// <summary>
         /// Returns an <see cref="CityResponse"/> for the specified ip address.
         /// </summary>
+        /// <param name="ip">The ip address.</param>
+        /// <returns>An <see cref="CityResponse"/></returns>
+        public CityResponse City(IPAddress ip)
+        {
+            return City(ip.ToString());
+        }
+
```
Comment (oschwald) [inline GeoIP2/WebServiceClient.cs]: Could you add an `Insights(IPAddress ip)` method as well?

**Comment by gturri** [comment]
Thanks for providing such a quick feedback!

Here is an update which takes your comments into account.

**Comment by oschwald** [comment]
Thanks! :+1:

**Pull request merged**